qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -152,10 +157,129 @@ void assertBasicSearchWorks(String index) throws IOException {
         assertEquals(numDocs1, numDocs2);
     }
 
+    void assertAllSearchWorks(String index) throws IOException {
+        logger.info("--> testing _all search");
+        Map<String, Object>  searchRsp = toMap(client().performRequest("GET", "/" + index + "/_search"));
+        assertNoFailures(searchRsp);
+        int totalHits = (int) XContentMapValues.extractValue("hits.total", searchRsp);
+        assertTrue("totalHits < " + totalHits, totalHits >=  1);
+        @SuppressWarnings("unchecked")
+        Map<String, Object> bestHit = (Map<String, Object>)
```

Comment:
`Map<?, ?> bestHit = (Map<?, ?>) ...` will probably work and won't require the SuppressWarnings.